### PR TITLE
Add comments for Ethereum governance contracts

### PR DESCRIPTION
**Pull request by claudijd**
This change is [Reviewable](https://reviewable.io/reviews/certusone/wormhole/1030)

### CODE REVIEW

File: ethereum/contracts/Governance.sol [hunk 1]
```diff
@@ -10,84 +10,137 @@ import "./Setters.sol";
 
 import "@openzeppelin/contracts/proxy/ERC1967/ERC1967Upgrade.sol";
 
+/**
+ * @dev `Governance` defines a means to enacting changes to contracts,
```
Comment (kcsongor) [inline ethereum/contracts/Governance.sol:14]: Specifically the core bridge contract

**Review by bruce-riley — APPROVED** [review]
> Reviewed 2 of 2 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/certusone/wormhole/1030)* status: all files reviewed, 3 unresolved discussions (waiting on @claudijd and @karlkempe)

**Review by karlkempe — DISMISSED** [review]

**Review by claudijd — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/certusone/wormhole/1030)* status: all files reviewed, 3 unresolved discussions (waiting on @kcsongor)

___
*[`ethereum/contracts/Governance.sol` line 14 at r1](https://reviewable.io/reviews/certusone/wormhole/1030#-MzCBWZ8yaAsUA1VhjXx-r1-14:-N0k3s56CUnIoPxNcmuy:b-896fix) ([raw file](https://github.com/certusone/wormhole/blob/ff9646093c760ea1bdde3a0a5d86ce3f35651113/ethereum/contracts/Governance.sol#L14)):*
<details><summary><i>Previously, kcsongor (Csongor Kiss) wrote…</i></summary><blockquote>

Specifically the core bridge contract
</blockquote></details>

Done.
___
*[`ethereum/contracts/Governance.sol` line 173 at r1](https://reviewable.io/reviews/certusone/wormhole/1030#-MzCBWZ8yaAsUA1VhjXx-r1-173:-N0k3sgk7BIuHq-ScIGC:b-896fix) ([raw file](https://github.com/certusone/wormhole/blob/ff9646093c760ea1bdde3a0a5d86ce3f35651113/ethereum/contracts/Governance.sol#L173)):*
<details><summary><i>Previously, kcsongor (Csongor Kiss) wrote…</i></summary><blockquote>

This verifies that the source chain is the governance chain (Solana)
</blockquote></details>

Done.
___
*[`ethereum/contracts/Governance.sol` line 178 at r1](https://reviewable.io/reviews/certusone/wormhole/1030#-MzCBWZ8yaAsUA1VhjXx-r1-178:-N0k3t2X0_j-U5E5HgjE:b-896fix) ([raw file](https://github.com/certusone/wormhole/blob/ff9646093c760ea1bdde3a0a5d86ce3f35651113/ethereum/contracts/Governance.sol#L178)):*
<details><summary><i>Previously, kcsongor (Csongor Kiss) wrote…</i></summary><blockquote>

Similarly, this verifies that the emitter contract is the governance contract (0x4 left padded)
</blockquote></details>

Done.

**Comment by claudijd** [comment]
@karlkempe @bruce-riley I made a couple tweaks to comments for @kcsongor, could you reapprove so I can merge?

**Review by bruce-riley — APPROVED** [review]

**Review by kcsongor — APPROVED** [review]

**Review by claudijd — COMMENTED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/certusone/wormhole/1030#-:-N2l-zdE6cbMzfT-OHG-:bnfp4nl)* status: 1 of 2 files reviewed, 3 unresolved discussions (waiting on @bruce-riley and @kcsongor)

**Review by claudijd — COMMENTED** [review]
> Dismissed @kcsongor from 3 discussions.
> *[Reviewable](https://reviewable.io/reviews/certusone/wormhole/1030)* status: 1 of 2 files reviewed, all discussions resolved (waiting on @bruce-riley)

**Review by claudijd — COMMENTED** [review]
> Reviewed 1 of 2 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/certusone/wormhole/1030)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @claudijd)

**Pull request closed without merging**